utils/mock-data-loader/insert-mocks-gef.js

```diff
@@ -24,7 +24,7 @@ const insertMocks = async (token) => {
 
   const makerToken = await portalApi.loginViaPortal(BANK1_MAKER1);
 
-  for (const [_index, item] of MOCKS.APPLICATION.entries()) {
+  for (const [item] of MOCKS.APPLICATION.entries()) {
```

Comment:
I think this may be a mistake as the first element of `.entries()` is the index of the array and the second element is the value of the element at that index. If `MOCKS.APPLICATION` is an array (which I think it must be if we are using `.entries()`) then this can just be `const item of MOCKS.APPLICATION` I think

Reply:
Ah! yes it is apologies for that, have fixed. Thank you for pointing it out.